backend/src/test/java/com/github/astridottmann/backend/controllers/CompareRoutesIntegrationTest.java

```diff
@@ -0,0 +1,199 @@
+                                    "fuel": "petrol",
+                                    "carSize": "large"
+                                },
+                                "co2EmissionRoute": 268.93
+                            },
+                            {
+                                "id": "456",
+                                "start": "Hamburg",
+                                "destination": "Frankfurt",
+                                "distance": 492,
+                                "numberOfPersons": 1,
+                                "oneWay": false,
+                                "vehicle": {
+                                    "type": "publicTransport",
+                                    "co2Emission": 46.0,
+                                    "distanceLevel": "longDistance",
+                                    "meansOfTransport": "train"
+                                },
+                                "co2EmissionRoute": 45.26
+                            }
+                        ]
+                """;
+
+        testCompareRoutesWithoutIdJson = """
+                 { "compared": [
+                            {
+                                "id": "123",
+                                "start": "Hamburg",
+                                "destination": "Frankfurt",
+                                "distance": 492,
+                                "numberOfPersons": 1,
+                                "oneWay": false,
+                                "vehicle": {
+                                    "type": "car",
+                                    "co2Emission": 253.3,
+                                    "fuel": "petrol",
+                                    "carSize": "large"
+                                },
+                                "co2EmissionRoute": 268.93
+                            },
+                            {
+                                "id": "456",
+                                "start": "Hamburg",
+                                "destination": "Frankfurt",
+                                "distance": 492,
+                                "numberOfPersons": 1,
+                                "oneWay": false,
+                                "vehicle": {
+                                    "type": "publicTransport",
+                                    "co2Emission": 46.0,
+                                    "distanceLevel": "longDistance",
+                                    "meansOfTransport": "train"
+                                },
+                                "co2EmissionRoute": 45.26
+                            }
+                        ],
+                     "comparisonResults": {
+                            "resultRouteOne": 268.93,
+                            "resultRouteTwo": 45.26,
+                            "difference": 223.67
+                        }
+                    }
+                """;
```

Comment:
Something like that should work:
```suggestion
        testCompareRoutesWithoutIdJson = objectMapper.writeValueAsString(testCompareRoutesWithoutId[1]);
```